geps/gep-1619.md

```diff
@@ -62,14 +62,110 @@ Header-based stateful sessions are achieved by a backend or gateway providing an
 
 Session information can be also encoded into the request URL to establish a persistent session. The server rewrites the client's URL to encode the new session information automatically. The server then decodes the session information from the URL to identify the session.
 
-**Session Initiation**
+### Session Persistence Initiation
 
-For both header-based and cookie-based sessions, either the gateway or the backend server can initiate establishing the session via the appropriate header or set-cookie attributes. The following rules apply based on who initiates the session:
+For both header-based and cookie-based sessions, the initiation of session persistence is possible from various sources,
+including the gateway, backend, a sidecar in a backend, or any other infrastructure component. The client application
+also plays in a role in session initiation. If a client doesn't want a request to belong an existing session, it can
+remove the session cookie or identifier to signal the gateway or backend to recreate the session.
```

Comment:
Not sure about this section - it would force an implementation to expose the details ( which cookie is used ? What if a cookie is used for multiple services, encrypted - and client is a browser ) ? 

I don't think it's a requirement for client to control the session - server should do it.